networkx/convert_matrix.py

```diff
@@ -348,7 +348,7 @@ def from_pandas_edgelist(
     edge_attr : str or int, iterable, True, or None
         A valid column name (str or int) or iterable of column names that are
         used to retrieve items and add them to the graph as edge attributes.
-        If `True`, all of the remaining columns will be added.
+        If `True`, all of the remaining columns will be added (not including `edge_key`).
```

Comment:
Is this better wording?  (it is <= 88 chars -- which is why I first started looking, but the end result is an attempt to allow the user to know what's going on without having to read all the other kwargs.
```suggestion
        If `True`, all columns will be added except `source`, `target` and `edge_key`.
```

Reply:
Looks better to me, thanks @dschult